rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java

```diff
@@ -77,9 +78,9 @@ public void stop() {
     }
 
     private void closeAll() {
-        logger.trace("closing blockStore.");
+        getLogger().trace("closing blockStore.");
```

Comment:
I don't think this is the right way to tackle this. Instead, I would change the logging level of the test logback.xml

Reply:
changed as suggested, thanks!